apps/src/netsim/NetSimRemoteNodeSelectionPanel.js

```diff
@@ -345,6 +360,23 @@ NetSimRemoteNodeSelectionPanel.prototype.shouldShowNode = function (node) {
 };
 
 /**
+ * @returns {boolean} TRUE if the current user is the only client node connected
+ *          to the shard right now.
+ */
+NetSimRemoteNodeSelectionPanel.prototype.isUserAlone = function () {
+  const otherUserCount = this.nodesOnShard_.filter(node =>
```

Comment:
could also use [`Array.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find), which has the minor advantage of returning as soon as a match is found.